JsonTabelViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Json+TabelViewController.swift
+//  new project 23.11.20
+//
+//  Created by Darya Gumus on 26.12.20.
+//  Copyright © 2020 Darya Gumus. All rights reserved.
+//
+
+import UIKit
+
+class JsonTabelViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
```

Comment:
Для теста ок, но с точки зрения перспективы следует называть объекты соответственно назначению
```suggestion
class UsersTabelViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
```